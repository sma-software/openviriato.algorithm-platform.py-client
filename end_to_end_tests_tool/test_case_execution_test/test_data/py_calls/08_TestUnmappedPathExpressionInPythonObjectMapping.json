{
  "PythonObjectMapping" : [
    {
      "ParameterName" : "routing_point",
      "ClassName": "RoutingPoint",
      "PythonParameterMapping": {
        "node_id" : "$.urlParameter.[0]"
      }
    }
  ]
}