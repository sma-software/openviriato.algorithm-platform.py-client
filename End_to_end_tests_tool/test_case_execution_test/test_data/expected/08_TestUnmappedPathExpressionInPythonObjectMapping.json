{
    "Integer": 1,
    "String": "2"
}